src/coreclr/vm/dllimport.cpp

```diff
@@ -2104,13 +2104,7 @@ void NDirectStubLinker::DoNDirect(ILCodeStream *pcsEmit, DWORD dwStubFlags, Meth
                 EmitLoadStubContext(pcsEmit, dwStubFlags);
                 // pcsEmit->EmitCALL(METHOD__STUBHELPERS__GET_NDIRECT_TARGET, 1, 1);
```

Comment:
```suggestion

                // NDirectMethodDesc::GetNDirectTarget
```
`METHOD__STUBHELPERS__GET_NDIRECT_TARGET` looks like another piece of dead code related to this. This can be deleted:
```
src\coreclr\System.Private.CoreLib\src\System\StubHelpers.cs(1117):internal static extern IntPtr GetNDirectTarget(IntPtr pMD);
src\coreclr\vm\corelib.h(952):DEFINE_METHOD(STUBHELPERS,          GET_NDIRECT_TARGET,     GetNDirectTarget,           SM_IntPtr_RetIntPtr)
src\coreclr\vm\ecalllist.h(621):FCFuncElement("GetNDirectTarget", StubHelpers::GetNDirectTarget)
src\coreclr\vm\stubhelpers.cpp(458):FCIMPL1(void*, StubHelpers::GetNDirectTarget, NDirectMethodDesc* pNMD)
src\coreclr\vm\stubhelpers.h(59):static FCDECL1(void*,           GetNDirectTarget,       NDirectMethodDesc* pNMD);
```

Reply:
Removing this from the ecalllist requires a JIT/EE version change, correct?

There's also on `getAddressOfPInvokeFixup` a dead `ppIndirection` parameter that we only ever assign `NULL` and assert is `NULL` on return.

Reply:
> Removing this from the ecalllist requires a JIT/EE version change, correct?

No. ecalllist is not part of JIT/EE contract.